youtube_dl_gui/locale/zh_CN/LC_MESSAGES/youtube_dl_gui.po

```diff
@@ -0,0 +1,442 @@
+msgid "Log Size: {0} Bytes"
+msgstr "日志大小：{0}字节"
+
+#: optionsframe.py:406 optionsframe.py:1425
+msgid "Restart"
+msgstr "重启"
+
+#: optionsframe.py:407
+msgid "Please restart {0}"
+msgstr "请重启{0}"
+
+#: optionsframe.py:517
+msgid "Shutdown when finished"
+msgstr "完成后关机"
+
+#: optionsframe.py:518
+msgid "SUDO password"
+msgstr "密码"
+
+#: optionsframe.py:570
+msgid "Playlist Start"
+msgstr "播放列表起始索引"
+
+#: optionsframe.py:571
+msgid "Playlist Stop"
+msgstr "播放列表结束索引"
+
+#: optionsframe.py:572
+msgid "Max Downloads"
+msgstr "最大下载数"
+
+#: optionsframe.py:630
+msgid "Retries"
+msgstr "重试次数"
+
+#: optionsframe.py:631
+msgid "User Agent"
+msgstr "用户代理"
+
+#: optionsframe.py:632
+msgid "Referer"
+msgstr "引用"
+
+#: optionsframe.py:633
+msgid "Proxy"
+msgstr "代理"
+
+#: optionsframe.py:707
+msgid "Username"
+msgstr "用户名"
+
+#: optionsframe.py:708
+msgid "Password"
+msgstr "密码"
+
+#: optionsframe.py:709
+msgid "Video Password (vimeo, smotri)"
+msgstr "视频密码（vimeo, smotri）"
+
+#: optionsframe.py:767
+msgid "high"
+msgstr "高"
+
+#: optionsframe.py:767
+msgid "low"
+msgstr "低"
+
+#: optionsframe.py:767
+msgid "mid"
+msgstr "中"
+
+#: optionsframe.py:770
+msgid "Convert to Audio"
+msgstr "音频转换"
+
+#: optionsframe.py:771
+msgid "Keep Video"
+msgstr "保留视频文件"
+
+#: optionsframe.py:772
+msgid "Audio Format"
+msgstr "音频格式"
+
+#: optionsframe.py:773
+msgid "Audio Quality"
+msgstr "音频质量"
+
+#: optionsframe.py:890
+msgid "default"
+msgstr "默认"
+
+#: optionsframe.py:891
+msgid "none"
+msgstr "无"
+
+#: optionsframe.py:895
+msgid "Video Format"
+msgstr "视频格式"
+
+#: optionsframe.py:896
+msgid "Mix Format"
+msgstr "混合格式"
+
+#: optionsframe.py:955
+msgid "Restrict filenames (ASCII)"
+msgstr "限定文件名（ASCII)"
+
+#: optionsframe.py:956
+msgid "ID as Name"
+msgstr "视频ID作为文件名"
+
+#: optionsframe.py:957
+msgid "Title as Name"
+msgstr "视频标题作为文件名"
+
+#: optionsframe.py:958
+msgid "Custom Template (youtube-dl)"
+msgstr "自定义文件名"
+
+#: optionsframe.py:1052
+msgid "Ignore Errors"
+msgstr "忽略错误"
+
+#: optionsframe.py:1053
+msgid "Open destination folder"
+msgstr "完成后打开下载目录"
+
+#: optionsframe.py:1054
+msgid "Write info to (.json) file"
+msgstr "保存视频元数据"
+
+#: optionsframe.py:1055
+msgid "Write description to file"
+msgstr "保存视频描述信息"
+
+#: optionsframe.py:1056
+msgid "Write thumbnail to disk"
+msgstr "保存视频缩略图"
+
+#: optionsframe.py:1057
+msgid "Filesize"
+msgstr "文件大小"
+
+#: optionsframe.py:1058
+msgid "Min"
+msgstr "最小"
+
+#: optionsframe.py:1059
+msgid "Max"
+msgstr "最大"
+
+#: optionsframe.py:1172
+msgid "English"
+msgstr "English"
+
+#: optionsframe.py:1173
+msgid "Greek"
+msgstr "Ελληνικά"
+
+#: optionsframe.py:1174
+msgid "Portuguese"
+msgstr "Português"
+
+#: optionsframe.py:1175
+msgid "French"
+msgstr "Français"
+
+#: optionsframe.py:1176
+msgid "Italian"
+msgstr "Italiano"
+
+#: optionsframe.py:1177
+msgid "Russian"
+msgstr "русский"
+
+#: optionsframe.py:1178
+msgid "Spanish"
+msgstr "Español"
+
+#: optionsframe.py:1179
+msgid "German"
+msgstr "Deutsch"
+
+#: optionsframe.py:1180
```

Comment:
Please remove lines 380 to 383. Those are for the available subtitles no for the available languages.